portalnet/src/find/iterators/findcontent.rs

```diff
@@ -602,6 +646,7 @@ mod tests {
                         assert_eq!(num_results, closest_nodes.len(), "Too  many results.");
                     }
                 }
+                _ => panic!("Unexpected result."),
```

Comment:
Since you mention "a crashed trin" makes me think that you might not have noticed this wildcard (and the earlier wildcard) are inside tests? When it comes to application code, I definitely agree that wildcards are poor patterns to rely upon, however when it comes to tests I tend to reach for them quite frequently (I'm sure I've left plenty around our codebase). It's less code clutter within each test and if the test fails I've found that it's typically not too complicated to hunt down the culprit.

Reply:
Yup, you're right. I missed that these were in tests!